File: front-end/app/models/Item.js
class Item {
  constructor(
    item_iD,
    item_name,
    category,
    description,
    amount_in_stock,
    price,
    futureAvailability,
    imagePath,
    //qty,
    brand
  ) {
    this.id = item_iD;
    this.name = item_name;
    this.category = category;
    this.description = description;
    this.stock = amount_in_stock;
    this.price = price;
    this.brand = brand;
    this.futureAvailability = futureAvailability;
    this.image = imagePath;
  }
}

export default Item;

File: Backend/src/main/java/Model/Item.java
package Model;
import java.awt.Image;
import java.sql.Date;



public class Item {
	private String id = null;
	private String itemName = null;
	private String image = null;
	private String category = null;
	private String description = null;
	private int currentQuantity = 0;
	private double price = 0.0;
	private Date futureAvailability = null;
	
	public Item() {
		
	}
	
	public String getID() {
		return id;
	}
	public void setID(String id) {
		this.id = id;
	}
	public String getItemName() {
		return itemName;
	}
	public void setItemName(String itemName) {
		this.itemName = itemName;
	}
	public String getImage() {
		return image;
	}
	public void setImage(String image) {
		this.image = image;
	}
	public String getCategory() {
		return category;
	}
	public void setCategory(String category) {
		this.category = category;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public int getCurrentQuantity() {
		return currentQuantity;
	}
	public void setCurrentQuantity(int currentQuantity) {
		this.currentQuantity = currentQuantity;
	}
	public double getPrice() {
		return price;
	}
	public void setPrice(double d) {
		this.price = d;
	}
	public Date getFutureAvailability() {
		return futureAvailability;
	}
	public void setFutureAvailability(Date futureAvailability) {
		this.futureAvailability = futureAvailability;
	}
	
	
}
